inferno-framework/tutorial-test-kit
# FI-3166: Step 3 solution upgrade

## Summary
This PR updates the Inferno template version in step-3-solution branch. It includes the solution to step-3 of the tutorial.

## Testing
Run locally with Docker:
- Run `./run.sh` in this repository
- Navigate to `http://localhost`
- Run test kit with the following inputs:
    - FHIR Base Server URL: `https://inferno.healthit.gov/reference-server/r4`
    - access_token: `SAMPLE_TOKEN`
    - Patient ID: `85`
- You should get the following results:

<img width="296" alt="image" src="https://github.com/user-attachments/assets/4ae0b8e9-f1af-44cc-bc3b-36951f06961d">

NOTE: The 2.02 Test fails because the Patients in the reference server no longer conform to USCore 3.1.1. This is expected.

Also run with Ruby.
